activerecord/test/cases/associations/required_test.rb

```diff
@@ -22,30 +22,6 @@ class Child < ActiveRecord::Base
     @connection.drop_table "children", if_exists: true
   end
 
-  test "belongs_to associations are not required by default" do
```

Comment:
Well.. I changed the test to

```
    model = subclass_of(Child) do
      belongs_to :parent, required: true, inverse_of: false,
        class_name: "RequiredAssociationsTest::Parent"
    end
```

And it fails... maybe I misunderstood something... Are not `belongs_to` associations required?